StreamNode/WebClient/package.json

```diff
@@ -33,7 +33,8 @@
     "postcss": "^7.0.35",
     "preact": "^10.5.10",
     "preact-render-to-string": "^5.1.12",
-    "preact-router": "^3.2.1"
+    "preact-router": "^3.2.1",
+    "react-responsive": "^8.2.0"
```

Comment:
We should remove react-responsive and use already implemented ant design responsive .